build/commands/lib/config.js

```diff
@@ -303,6 +304,19 @@ Config.prototype.buildArgs = function () {
 
   if (!this.isBraveReleaseBuild()) {
     args.chrome_pgo_phase = 0
+
+    if (process.platform === 'darwin' && this.targetOS != 'ios' && !this.shouldSign()) {
+      // Currently we're using is_official_build mode in PR builds on CI. This enables dSYMs
+      // by default, which slows down link phase, but also disables relocatable compilation 
+      // on MacOS (aka 'zero goma cachehits' style).
+      //
+      // Don't create dSYMs in non-public Release builds and don't strip debug info.
+      // See //build/config/apple/symbols.gni for additional details.
+      if (!this.isAsan()) {
+        args.enable_dsyms = false
```

Comment:
enable_dsyms should already be false `enable_dsyms = is_official_build || using_sanitizer`

Reply:
sorry, it's `||`, I misread that